src/models/UserMapper.php

```diff
@@ -544,6 +544,31 @@ public function getUserIdFromUsername($username)
     }
 
     /**
+     * Function to get just the user ID
+     *
+     * @param string $fullname The users full name we're looking for
+     *
+     * @return $user_id The user's ID (or false, if we didn't find her)
+     */
+    public function getUserIdFromFullname($fullname)
```

Comment:
It actually works that way that I can add a user as host by it's name. I'll still need to dig into the code-portion of that though. Tonight looks promising...